preview/_base.css

```
@@ -721,42 +721,40 @@ a.no-link::after { display: none; }
-}
-.struck[data-strike="draw"]::after {
-  content: "";
-  position: absolute;
-  left: 0; right: 0;
-  top: 56%;
-  height: 2px;
-  background: var(--red);
-  transform: scaleX(0);
-  transform-origin: left center;
-  transition: transform var(--dur-slow) var(--ease-out-quart);
+  text-decoration: line-through;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)